cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/KafkaSampleStore.java

```diff
@@ -215,16 +265,11 @@ public void configure(Map<String, ?> config) {
 
   @SuppressWarnings("unchecked")
   protected void ensureTopicsCreated(Map<String, ?> config) {
-    String connectString = (String) config.get(ExecutorConfig.ZOOKEEPER_CONNECT_CONFIG);
-    boolean zkSecurityEnabled = (Boolean) config.get(ExecutorConfig.ZOOKEEPER_SECURITY_ENABLED_CONFIG);
-    KafkaZkClient kafkaZkClient = KafkaCruiseControlUtils.createKafkaZkClient(connectString,
-                                                                              "KafkaSampleStore",
-                                                                              "EnsureTopicCreated",
-                                                                              zkSecurityEnabled);
-    AdminZkClient adminZkClient = new AdminZkClient(kafkaZkClient);
     AdminClient adminClient = KafkaCruiseControlUtils.createAdminClient((Map<String, Object>) config);
     try {
-      Map<String, List<PartitionInfo>> topics = _consumers.get(0).listTopics();
+      short replicationFactor = sampleStoreTopicReplicationFactor(config, adminClient);
```

Comment:
Looks like `sampleStoreTopicReplicationFactor(config, adminClient)` could return null here, will this be an issue in unboxing?